src/bash/include.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+gzdoom="gzdoom" #"/path/to/gzdoom"
```

Comment:
This line is better off in a separate file. I suggest `settings.sh`. This would mirror the idea of [settings.ps1](https://github.com/borogk/zdoom-cameraman/blob/main/src/ps1/settings.ps1), which is the only file a user is supposed to modify at all. 

Other than that, the idea of this separate module is solid.